src/main/java/edu/cornell/kfs/fp/document/CuDisbursementVoucherDocument.java

```diff
@@ -923,6 +924,48 @@ protected boolean isCampusReviewRequired() {
 
         return false;
     }
+    
+    @Override
+    protected boolean isTaxReviewRequired() {
```

Comment:
Thank you for pointing this out Bryan. Unfortunately I could not find a way to call super and eliminate duplication. I know this will make it difficult when we upgrade. If this method changes in foundation code it can be tricky to determine what was Cornell customization and what not, and the developer doing the upgrade would need to understand what changed in foundation code between different versions and what part was the intended Cornell customization. I thought about adding a comment surrounding the part of the code that was added for this change but comments can also be tricky. If any other developer can suggest a better way to implement this that would be greatly appreciated!